babel/messages/pofile.py

```diff
@@ -276,11 +283,13 @@ def parse(self, fileobj):
             self._add_message()
 
     def _invalid_pofile(self, line, lineno, msg):
+        if self.abort_invalid:
+            raise PoFileError("Invalid POFile encountered line_number={}, msg={}".format(lineno, msg))
```

Comment:
See the discussion above about a better exception :)

Reply:
👍